code/modules/mob/living/carbon/human/life.dm

```diff
@@ -366,12 +366,11 @@ All effects don't start immediately, but rather get worse over time; the rate is
 			blur_eyes(5)
 
 		if(drunkenness >= 81)
-			adjustToxLoss(0.2)
+			adjustToxLoss(0.1)
 			if(prob(5) && !stat)
 				src << "<span class='warning'>Maybe you should lie down for a bit...</span>"
 
 		if(drunkenness >= 91)
-			adjustBrainLoss(0.4)
```

Comment:
I agree, the brain loss needs a nerf, but leave it.  Perhaps make it only give up to 50 brainloss?